.github/labeler.yml

```diff
@@ -0,0 +1,20 @@
+version: 1
+appendOnly: true
+# Labels are applied based on conventional commits standard
+# https://www.conventionalcommits.org/en/v1.0.0/
+# These labels are later used in release notes. See .github/release.yml
+labels:
+# If the PR title has an ! before the : it will be considered a breaking change
+# For example, `feat!: add new feature` will be considered a breaking change
```

Comment:
I think conventional commits says "either do the ! OR add BREAKING CHANGE in the body (and both is ok)".  I think this is fine, I'll just need to remember to check for the !.

Reply:
Added a condition for `BREAKING CHANGE` in body.